README.md

```diff
@@ -1,2 +1,50 @@
-# DevOps-Azure-plugin
-Leapwork Integration with DevOps Azure
+# Leapwork Integration  
+Easily include Leapwork as part of your AzureDevops pipeline using Leapwork Azure devops extension
+
+# More Details 
+Leapwork’s completely visual, no-code automation platform makes it easy for business and IT users to automate repetitive processes, so enterprises can adopt and scale automation faster. For rapid results, at lower cost, and requiring fewer specialist resources than traditional automation approaches. 
+
+# Features:
```

Comment:
Make sure Features are presented in list, because in your branch it looks like one line.
![image](https://user-images.githubusercontent.com/72276668/132822117-74f3b21c-e0e7-4b77-b96d-450ef5df70da.png)

Reply:
Done, with readme file